middleware.cfg

```diff
@@ -40,7 +40,7 @@
 180:file:file
 190:auto:auto
 200:secondary:secondary
-210:etcd:etcd
+210:etcd:github.com/coredns/coredns/middleware/backend/etcdv2
```

Comment:
If need be, I'll modify the generator to support nested folders. In the mean time, I really appreciate your feedback and will keep updating the PR.